### PR TITLE
Filter validate get update

### PR DESCRIPTION
**Pull request by nicky-johnstone**
Tcl
----
- [x] Does each proc have a #| comment to describe what it does ?
- [x] Are there comments throughout the code ?
- [x] Are variable names well chosen ?
- [x] List what testing has been done.
  - Tested on MLA ERP DEV:
  - `/sales_order_select.html` form - entering invalid and valid data.
  - Entering invalid values for query string params in URLs i.e. `/sales_order.html?sales_order_id=foo`

### CODE REVIEW

File: tcl/html.tcl [hunk 1]
```diff
@@ -869,3 +869,8 @@ proc qc::safe_attributes {} {
 
     return $safe_attributes
 }
+
+proc qc::global_response2html {} {
```
Comment (bvw) [inline tcl/html.tcl]: what about `qc::response2html`

Comment (nicky-johnstone) [inline tcl/html.tcl]: Considered that but didn't know if it was a little bit too generic like `data2html`.

**Comment by nicky-johnstone** [comment]
Updated.

**Comment by nicky-johnstone** [comment]
Updated.

**Pull request merged**